etc/scripts/build_c0.bash

```diff
@@ -0,0 +1,43 @@
+#! /bin/bash
+#
+# build_c0.bash fakes a series of agent tasks to build a C0
+#
+# Copyright (C) 2022 Matteo Tschesche <matteo.tschesche@rwth-aachen.de>
+#
+#  This program is free software; you can redistribute it and/or modify
+#  it under the terms of the GNU General Public License as published by
+#  the Free Software Foundation; either version 2 of the License, or
+#  (at your option) any later version.
+#
+#  This program is distributed in the hope that it will be useful,
+#  but WITHOUT ANY WARRANTY; without even the implied warranty of
+#  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#  GNU Library General Public License for more details.
+#
+#  Read the full text in the LICENSE.GPL file in the doc directory.
+
+#prepare cap
+./llsf-fake-robot bot-1 Carologistics 1 3 NONE NONE NONE NONE NONE Move 1 C-CS1 INPUT & sleep 10 ; kill $!
```

Comment:
It would be nice to give an initial description of what is happening here (Beacon Signals, Agent Tasks, etc.) so that this would be even more educational!